config/schedule.rb

```diff
@@ -0,0 +1,5 @@
+require File.expand_path(File.join(File.dirname(__FILE__), "..", "config", "environment"))
+
+every 1.day, at: "00:00" do
```

Comment:
Have you tested if its working - by running it on review app every 10 minutes or so? Ideally changing manually `ecf` DB?

Reply:
Yea, used the above mentioned approach and the logs stated that the job ran Ok. I just merged the refactored ECF code changes and I'm going to run a sanity on this again just to be sure.